state-chain/traits/src/mocks/price_feed_api.rs

```diff
@@ -29,4 +29,27 @@ impl PriceFeedApi for MockPriceFeedApi {
 		Self::get_storage::<_, Option<Price>>(ORACLE_PRICE, asset)
 			.and_then(|price| price.map(|price| OraclePrice { price, stale }))
 	}
+	fn get_relative_price(asset1: Asset, asset2: Asset) -> Option<OraclePrice> {
```

Comment:
You should be able to implement this directly on `PriceFeedApi` trait by means of calling `Self::get_price` (as the "default" implementation). Then implementors of `PriceFeedApi` (like this mock) won't need to duplicate this code.